src/modules/Dropdown/Dropdown.js

```diff
@@ -409,8 +418,9 @@ export default class Dropdown extends Component {
     } else if (prevState.focus && !this.state.focus) {
       debug('dropdown blurred')
       if (!this.isMouseDown) {
```

Comment:
Hey guys I know I'm late to the party. I wanted to discuss this line with you. So I'm doing a **search** Dropdown and I don't want it to open on focus. This works fine while tabbing through the form, but clicking on a form field is also focusing on it, yet it'll still open. 

It makes sense that the menu opens if you click on the arrow, but I think for search it's a bit unintuitive if it opens when one clicks on the input.

Thoughts?